Amethyst/Managers/WindowManager.swift

```diff
@@ -334,22 +298,20 @@ private struct ObserveApplicationNotifications {
                 return
             }
 
-            guard let resizedWindow = element as? SIWindow else {
+            guard let screen = window.screen(), windowManager.activeWindows(on: screen).contains(window) else {
                 return
             }
 
-            guard let screen = resizedWindow.screen(),
-                windowManager.activeWindows(on: screen).contains(resizedWindow) else {
-                    return
-            }
-
-            guard let screenManager = windowManager.focusedScreenManager(),
-                let layout = screenManager.currentLayout as? Layout & PanedLayout,
-                let oldFrame = layout.assignedFrame(resizedWindow, of: windowManager.activeWindowsForScreenManager(screenManager), on: screen) else {
-                    return
+            guard
```

Comment:
This conditional could benefit from a comment and/or a slight cleanup.  (Maybe some optional chaining?)